### DISCUSSION TITLE
Is this compatible with java17?

### DISCUSSION THREAD

**Discussion by siwonred** [xvik/dropwizard-guicey#201] [General]
Dropwizard seems to compatible with java17
Is this project compatible with 17 officially?
If not, can I know the plan?

**Comment by xvik** [comment]
In theory, the latest version should be. As I understand some people already using it with java 17.
To be 100% sure I need to enable java 17 for CI test, but I can't until I get rid of spock 1 (and that's why I'm working on spock 2 support now). I think will be able to finish it in January.

There seem to be problems when using modules path instead of classpath: #187. Of course, I'll fix this exact point, but not sure it's the only one (need time to investigate all this).

In any case, in case of urgent problems, I can always release a hotfix version (I just need to know about it :) ).
